src/main/java/com/minecolonies/commands/ColonyCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.minecolonies.commands;
+
+import com.google.common.collect.ImmutableMap;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+/**
+ * Minecolonies root command.
+ * <p>
+ * Manages all sub commands.
+ */
+public class ColonyCommand extends AbstractSplitCommand
+{
+
+    public static final String DESC = "colony";
+
+    private final ImmutableMap<String, ISubCommand> subCommands =
+      new ImmutableMap.Builder<String, ISubCommand>()
+        .put("kill", new KillCitizen(MinecoloniesCommand.DESC, ColonyCommand.DESC, "kill"))
+        .put("respawn", new RespawnCitizen(MinecoloniesCommand.DESC, ColonyCommand.DESC, "respawn"))
+        .build();
+
+    /**
+     * Initialize this command with it's parents.
+     */
+    public ColonyCommand(@NotNull final String parent)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): parent [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)